### ISSUE TITLE
systemd unit file in latest (0.12.3-2) nomad package for Ubuntu (focal): should be s/StartLimitInterval/StartLimitIntervalSec/ ?

### ISSUE THREAD

**Issue by jf** [hashicorp/nomad#8783]
As per subject, if you install the latest nomad package for Ubuntu (following the instructions at https://www.nomadproject.io/docs/install; I'm using Focal), the systemd unit file is as follows:

```
     1  [Unit]
     2  Description=Nomad
     3  Documentation=https://nomadproject.io/docs/
     4  Wants=network-online.target
     5  After=network-online.target
     6
     7  # When using Nomad with Consul it is not necessary to start Consul first. These
     8  # lines start Consul before Nomad as an optimization to avoid Nomad logging
     9  # that Consul is unavailable at startup.
    10  #Wants=consul.service
    11  #After=consul.service
    12
    13  [Service]
    14  ExecReload=/bin/kill -HUP $MAINPID
    15  ExecStart=/usr/bin/nomad agent -config /etc/nomad.d
    16  KillMode=process
    17  KillSignal=SIGINT
    18  LimitNOFILE=65536
    19  LimitNPROC=infinity
    20  Restart=on-failure
    21  RestartSec=2
    22  StartLimitBurst=3
    23  StartLimitInterval=10
    24  TasksMax=infinity
    25  OOMScoreAdjust=-1000
    26
    27  [Install]
    28  WantedBy=multi-user.target
```

It looks like it should be `StartLimitIntervalSec=10` instead of `StartLimitInterval=10` at line 23

**Comment by notnoop** [comment]
@jf Good catch - Thank you very much!  I made a PR for the internal repository managing our Linux packaging.  I'll close the ticket when it's shipped.

**Comment by analytically** [comment]
This will produce a warning on older systemd versions.

**Comment by m1keil** [comment]
`StartLimitIntervalSec` also needs to be under [Unit] and not [Service] or else the following warning is showed:

```
/lib/systemd/system/nomad.service:16: Unknown key name 'StartLimitIntervalSec' in section 'Service', ignoring.
```

**Comment by tgross** [comment]
We've merged the change to packaging. That missed the ship window for Nomad 1.1.0 but will be available in the packages for Nomad 1.1.1.
